interface/main/tabs/js/tabs_view_model.js

```diff
@@ -100,6 +100,15 @@ function tabRefresh(data,evt)
     }
 }
 
+function tabRefreshByName(name) {
```

Comment:
recommend a comment here that this function is used to support custom/extended code so we don't remove it during code cleanups (where we remove code that appears unused)